training-node/app/services/tokenManager.js

```diff
@@ -1,10 +1,16 @@
 const jwt = require('jwt-simple'),
-  config = require('../../config');
+  config = require('../../config'),
+  logger = require('../logger');
 
 const SECRET = config.common.session.secret;
 
 exports.HEADER_NAME = config.common.session.header_name;
 
-exports.encode = payload => jwt.encode(payload, SECRET);
+exports.encode = email => {
+  const expiration = parseInt(config.common.session.expiration);
+  const nbf = parseInt(Date.now() / 1000);
```

Comment:
use moment here

Reply:
what is nbf?

Reply:
nbf is identifies the time from which token is accepted

Reply:
oh i didnt know that definition